lascli/__version__.py

```diff
@@ -7,4 +7,4 @@
 __maintainer_email__ = 'magnus@lucidtech.ai'
 __title__ = 'lucidtech-las-cli'
 __url__ = 'https://github.com/LucidtechAI/las-cli'
-__version__ = '9.2.2'
```

Comment:
I believe we have som breaking changes here since remote_component_asset_id argument has been removed, so I think bumping up to 10.0.0 would be correct.